### PR TITLE
Lateral padding of PageContainer

### PR DESCRIPTION
**Pull request by Thiago-spart**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] I followed GitFlow pattern to create the branch
- [x] My code produces no warnings or errors in the console
- [x] Tests for the changes have been added / updated
- [x] Docs have been added / updated

## PR Type

What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[x] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Other... Please describe:
```

## What is the current behavior?
The old media querie was not necessary
## What is the new behavior?
Now instead of querie we have a padding in the links
## Does this PR introduce a breaking change?

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by henriqueleite42 — DISMISSED** [review]

File: src/web/pages/Home/components/MainSection/styles.ts [hunk 1]
```diff
@@ -39,6 +39,7 @@ const RedirectCSS = css`
 
 export const Links = styled.div`
 	display: flex;
+	padding: 0 2rem;
```
Comment (alessandrohenriqueramos) [inline src/web/pages/Home/components/MainSection/styles.ts:42]: É pra alterar no `PageContainer`, defazer todas essas alterações

Comment (alessandrohenriqueramos) [inline src/web/pages/Home/components/MainSection/styles.ts:42]: É pra alterar no `PageContainer`, desfazer todas essas alterações

**Review by aleessandrohr — CHANGES_REQUESTED** [review]

File: src/web/pages/Home/components/MainSection/styles.ts [hunk 2]
```diff
@@ -52,7 +52,7 @@ export const Links = styled.div`
 		}
 
 		@media (max-width: 768px) {
-			margin-bottom: 2rem;
+			padding-top: 2rem;
```
Comment (alessandrohenriqueramos) [inline src/web/pages/Home/components/MainSection/styles.ts:55]: desfazer alteração

**Review by aleessandrohr — CHANGES_REQUESTED** [review]

File: src/assets/colors.ts [hunk 3]
```diff
@@ -1,4 +1,4 @@
-import { Colors as CoreColors } from "core/assets/colors";
+import { COLORS as CoreColors } from "core/assets/colors";
```
Comment (henriqueleite42) [inline src/assets/colors.ts:1]: `CoreColors` -> `CORE_COLORS`

**Review by henriqueleite42 — CHANGES_REQUESTED** [review]

**Review by henriqueleite42 — DISMISSED** [review]

**Review by aleessandrohr — APPROVED** [review]

**Pull request merged**